src/components/layout/PetCard.css

```diff
@@ -48,7 +48,8 @@
 
 .card__img {
   object-fit: cover;
-  height: 300px;
+  height: 50%;
+  width: 50%;
```

Comment:
lets not change this css maybe just create a new one specifically for the user story cards

Reply:
solved!